encord/objects/classification_instance.py

```diff
@@ -166,6 +166,17 @@ def set_for_frames(
             self._parent._add_frames_to_classification(self.ontology_item, frames_list)
             self._parent._add_to_frame_to_hashes_map(self, frames_list)
 
+    def set_frame_data(self, frame_data: FrameData, frames: Frames) -> None:
```

Comment:
We already create the frame_data object from the json dict we have. Instead of creating a new one, just set this one.